Intersections_3/include/CGAL/Intersections_3/internal/intersection_3_1_impl.h

```diff
@@ -79,6 +79,33 @@ do_intersect(const Plane_3<R> &plane1, const Plane_3<R> &plane2,
 namespace Intersections {
 namespace internal {
 
+template <class K>
+boost::optional<typename K::Point_3>
+intersection_point(const typename K::Plane_3 &plane,
+                   const typename K::Line_3 &line,
+                   const K& /*k*/)
+{
+  typedef typename K::Point_3 Point_3;
+  typedef typename K::Direction_3 Direction_3;
+  typedef typename K::RT RT;
+
+  const Point_3 &line_pt = line.point();
```

Comment:
`k` should be used for `k.construct_point_object()` etc...

Reply:
Note that it is like this all over the place in this file.